indico/web/client/js/react/components/MarkdownEditor.jsx

```diff
@@ -54,10 +62,16 @@ export default function MarkdownEditor({height, imageUploadURL, ...rest}) {
     }
   };
 
-  return (
+  return isLoadingRules ? (
+    <Loader />
+  ) : (
     <div styleName="markdown-editor" onDragOver={handleDragOver}>
       <MdEditor
-        renderHTML={text => <Markdown>{text.replace(/\n/gi, '  \n')}</Markdown>}
+        renderHTML={text => (
+          <Markdown remarkPlugins={[[AutoLinkerPlugin, {rules: data.rules}]]}>
```

Comment:
Probably good to add `targetBlank` to the props here so any links from the preview open in a new tab

Reply:
Jolly good idea.